### PR TITLE
One fix, and one variable rename, and one unfix'd issue; tbc.

### PR DESCRIPTION
**Pull request by myFree**

### CODE REVIEW

**Review by Fl4m3Ph03n1x — CHANGES_REQUESTED** [review]
Do we have a test that covers this change to the if statement?

File: darwin/dataset/remote_dataset.py [hunk 1]
```diff
@@ -387,7 +387,10 @@ def add_annotation_class(self, annotation_class):
             cls
             for cls in all_classes
             if cls["name"] == annotation_class.name
```
Comment (Fl4m3Ph03n1x) [inline darwin/dataset/remote_dataset.py:389]: Might be a good idea to put this if condition inside a function that makes it clear that it does (with the correct name) to go along the lines of the Python Good code practices.

File: darwin/importer/importer.py [hunk 2]
```diff
@@ -75,26 +75,26 @@ def get_remote_files(dataset, filenames):
     return remote_files
 
 
-def _resolve_annotation_classes(annotation_classes: List[dt.AnnotationClass], classes_in_dataset, classes_in_team):
+def _resolve_annotation_classes(local_annotation_classes: List[dt.AnnotationClass], classes_in_dataset, classes_in_team):
     local_classes_not_in_dataset = set()
```
Comment (Fl4m3Ph03n1x) [inline darwin/importer/importer.py:79]:
```
local_classes_not_in_dataset: Set[dt.AnnotationClass] = set()
```

File: darwin/importer/importer.py [hunk 3]
```diff
@@ -75,26 +75,26 @@ def get_remote_files(dataset, filenames):
...
     local_classes_not_in_team = set()
```
Comment (Fl4m3Ph03n1x) [inline darwin/importer/importer.py:80]: Same as above.

**Review by Fl4m3Ph03n1x — CHANGES_REQUESTED** [review]

File: darwin/dataset/remote_dataset.py (hunk 1 above)
Comment (myFree) [reply darwin/dataset/remote_dataset.py:389]: Could you share an example code snippet you feel would make this line clearer?

To me; putting it in a function would require going into that function to see what it does; which is an extra step when trying to understand the code.
That line of code only does one very obvious thing - it compares two values.

Comment (Fl4m3Ph03n1x) [reply darwin/dataset/remote_dataset.py:389]: I see you are comparing values, but I dont understand what is the semantic meaning you are trying to convey. 
Are you trying to check the class is the same or is compatible?

Comment (myFree) [reply darwin/dataset/remote_dataset.py:389]: Discussed on a productive call; understand Pedro's input - leaving as is for now

**Review by simedw — APPROVED** [review]

**Pull request merged**